src/app/components/shell/header/header.hbs

```diff
@@ -1,56 +1,78 @@
-            <span></span>
-          </button>
-        </div>
-      </nav>
+        <ul class="nav-menu">
+            <li class="nav-menu-item dropdown" role="presentation">
+                <a href="/subjects" role="menuitem" aria-haspopup="true">
+                    Subjects <span class="caret"></span>
+                </a>
+                <ul class="dropdown-menu" role="menu" aria-expanded="false" aria-label="Subjects menu">
+                    <li role="presentation"><a href="/subjects" role="menuitem" tabindex="-1">All</a></li>
+                    <li role="presentation"><a href="/subjects/math" role="menuitem" tabindex="-1">Math</a></li>
+                    <li role="presentation"><a href="/subjects/science" role="menuitem" tabindex="-1">Science</a></li>
+                    <li role="presentation"><a href="/subjects/social-sciences" role="menuitem" tabindex="-1">Social Sciences</a></li>
+                    <li role="presentation"><a href="/subjects/history" role="menuitem" tabindex="-1">History</a></li>
+                    <li role="presentation"><a href="/subjects/ap" role="menuitem" tabindex="-1">AP</a></li>
+                </ul>
+            </li>
+            <li class="nav-menu-item" role="presentation">
+                <a href="/higher-ed" role="menuitem">Higher Education</a>
+            </li>
+            <li class="nav-menu-item" role="presentation">
+                <a href="/k-12" role="menuitem">K-12</a>
+            </li>
+            <li class="nav-menu-item dropdown" role="presentation">
+                <a href="/impact" role="menuitem" aria-haspopup="true">
+                    Our Impact <span class="caret"></span>
+                </a>
+                <ul class="dropdown-menu" role="menu" aria-expanded="false" aria-label="Our Impact menu">
+                    <li><a href="/stories" role="menuitem" tabindex="-1">Success Stories</a></li>
+                    <li><a href="/team" role="menuitem" tabindex="-1">Our Team</a></li>
+                    <li><a href="/allies" role="menuitem" tabindex="-1">Allies</a></li>
+                    <li><a href="/careers" role="menuitem" tabindex="-1">Careers</a></li>
+                </ul>
+            </li>
+            <li class="nav-menu-item expand" role="menuitem" aria-haspopup="true" aria-label="Toggle Meta Navigation Menu" tabindex="0">
+                <span class="cls"></span>
+                <span>
+                    <ul class="sub-menu" role="menu" aria-expanded="false" aria-label="Meta navigation menu">
+                        <li><a href="/news" role="menuitem">News</a></li>
+                        <li><a href="/contact" role="menuitem">Contact</a></li>
+                        <li><a href="/give" role="menuitem">Give</a></li>
+                        <li><a href="/login" role="menuitem">Login</a></li>
+                    </ul>
+                </span>
+                <span class="cls"></span>
+            </li>
+        </ul>
+    </nav>
 </header>
```

Comment:
Im not sure why you are rewriting this html for the header? it seems what you are rewriting has nothing to do with the home page functionality. This is all relative to the header alone and I am responsible for this code. Please revert.